### DISCUSSION TITLE
Support for two-column conference paper pdfs similar to rticle's IEEE Transactions format?

### DISCUSSION THREAD

**Discussion by dbleyl** [quarto-dev/quarto-cli#2361] [Q&A]
Is there a template that can generate a conference paper pdf similar to the [rticle IEEE Transactions example](https://pkgs.rstudio.com/rticles/articles/examples.html#ieee---ieee-transactions-journal-article)?

Thanks,
Don

**Comment by mcanouil** [comment]
Far as I know[^1], this one does not exist yet.
But you could make that one following already published formats (<https://github.com/quarto-journals>) and documentation (<https://quarto.org/docs/journals/>).

[^1]: <https://github.com/mcanouil/awesome-quarto#templates>

**Reply by dbleyl** [reply]
OK, challenge accepted. Looks like there are many unfamiliar parts, so it will take some time.

**Reply by mcanouil** [reply]
You can start with the template: <https://github.com/quarto-journals/article-format-template> 😉
